### ISSUE TITLE
Support keyboard navigation

### ISSUE THREAD

**Issue by harry75369** [verygoodgraphics/vgg_runtime#382]
### Detailed description

For accessibility. If there is no mouse, and keyboard could be used to navigate inside a VGG frame.

### Possible implementation

- Element focusing should be implemented first
- Then `tab` event should be handled to iterate over focus-able elements
